src/dpi_transform.erl

```diff
@@ -81,55 +138,123 @@ nif_exports(NifDefns, Line) ->
 
 nif_exports([], Line, NewNifDefns) ->
     {attribute, Line, export, NewNifDefns};
-nif_exports([{Fun, FunNif, Arity, _Args} | NifDefns],
-        Line, NewNifDefns) ->
+nif_exports([{Fun, FunNif, Arity, _Args, _Mode}
+	     | NifDefns],
+	    Line, NewNifDefns) ->
     nif_exports(NifDefns, Line,
-        [{Fun, Arity}, {FunNif, Arity} | NewNifDefns]).
+		[{Fun, Arity}, {FunNif, Arity},
+		 {list_to_atom(atom_to_list(Fun) ++ "Gen"), Arity},
+		 {list_to_atom(atom_to_list(Fun) ++ "RPC"), Arity}
+		 | NewNifDefns]).
 
 nif_stubs([], _Line) -> [];
-nif_stubs([{Fun, FunNif, Arity, Args} | Rest], Line) ->
+nif_stubs([{Fun, FunNif, Arity, Args, Mode} | Rest],
+	  Line) ->
     FunParams = params(Args, Line + 1),
     FunGuards = guards(Args, FunParams, Line + 1),
     FunNifParams = params(Args, Line + 3),
     FunNifGuards = guards(Args, FunNifParams, Line + 3),
-    ?N("function ~p\n",[FunNif]),
+    ?N("function ~p\n", [FunNif]),
     [% -spec(Fun(Arg, ...) -> term()).
      {attribute, Line, spec,
```

Comment:
1. Add a new line before every PT section
1. Comment the resulting erlang pseudocode from each pt block
1. Indent multiline terms (lists/tuple) with 4 spaces